students/eowyn/session13/timer.py

```diff
@@ -0,0 +1,45 @@
+"""
+Record how long something takes to run. Print result and write to file,
+if a file object is provided. Otherwise print to stdout (screen). An
+optional name argument allows for the function being timed to be
+given a name which is logged.
+"""
+
+import sys
+import contextlib
+
+
+class Timer():
+    def __init__(self, fileobj=sys.stdout, name=""):
+        if name:
+            self.name = name
+        else:
+            self.name = "This code"
+        self.fileobj = fileobj
+
+    def __enter__(self):
+        from time import clock
```

Comment:
you want this import at the top of the file.

then it'll be  global and can be accessed everywhere.